wapi/util.py

```diff
@@ -104,7 +107,8 @@ def to_pandas(self, name=None):
         for row in self.points:
             if len(row) != 2:
                 raise ValueError('Points have unexpected contents')
-            index.append(datetime.datetime.fromtimestamp(row[0]/1000.0, self.tz))
+            dt = datetime.fromtimestamp(row[0] / 1000.0, self.tz)
+            index.append(dt)
```

Comment:
Igjen en endring for endringens skyld?  `dt` brukes ikke videre, så dette tar marginalt lenger tid å kjøre og gir hva?

Reply:
Det gir mer lesbar kode?

Reply:
Greit nok.